### PR TITLE
VZ-7397  Remove any remaining doc references to ElasticSearch/Kibana

### PR DESCRIPTION
**Pull request by knelson7**

### CODE REVIEW

**Review by anders-swanson — APPROVED** [review]

File: content/en/docs/reference/API/Verrazzano/v1beta1.md [hunk 1]
```diff
@@ -165,15 +165,15 @@ spec:
...
+| `opensearchSecret` | string | The secret containing the credentials for connecting to OpenSearch.  This secret needs to be created in the `verrazzano-install` namespace prior to creating the Verrazzano custom resource.  Specify the OpenSearch login credentials in the `username` and `password` fields in this secret.  Specify the CA for verifying the OpenSearch certificate in the `ca-bundle` field, if applicable.  The default `verrazzano` is the secret for connecting to the VMI OpenSearch.                      | No |
+| `oci` | [OCILoggingConfiguration](#oci-logging-configuration) | The Oracle Cloud Infrastructure Logging configuration.                                                                                                                                                                                                                                                                                                                                                                                                                                              | No |
```
Comment (rosemarymarano) [inline content/en/docs/reference/API/Verrazzano/v1beta1.md:174]: This link doesn't work [OCILoggingConfiguration](#oci-logging-configuration) -> It should be [OCILoggingConfiguration](#oracle-cloud-infrastructure-logging-configuration) (The heading is Oracle Cloud Infrastructure Logging Configuration)

**Review by rosemarymarano — COMMENTED** [review]
One minor edit.

File: content/en/docs/customize/opensearch.md [hunk 2]
```diff
@@ -71,9 +71,9 @@ $ kubectl get pvc,pod -l verrazzano-component=opensearch -n verrazzano-system
 
 # Sample output
 NAME                                                             STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
-persistentvolumeclaim/elasticsearch-master-vmi-system-master-0   Bound    pvc-9ace042a-dd68-4975-816d-f2ca0dc4d9d8   50Gi       RWO            standard       5m22s
-persistentvolumeclaim/elasticsearch-master-vmi-system-master-1   Bound    pvc-8bf68c2c-235e-4bd5-8741-5a5cd3453934   50Gi       RWO            standard       5m21s
-persistentvolumeclaim/elasticsearch-master-vmi-system-master-2   Bound    pvc-da8a48b1-5762-4669-98f0-8479f30043fc   50Gi       RWO            standard       5m21s
+persistentvolumeclaim/opensearch-master-vmi-system-master-0   Bound    pvc-9ace042a-dd68-4975-816d-f2ca0dc4d9d8   50Gi       RWO            standard       5m22s
```
Comment (anders-swanson) [inline content/en/docs/customize/opensearch.md:74]: are these still column aligned?

File: content/en/docs/customize/opensearch.md [hunk 3]
```diff
@@ -91,7 +91,7 @@ pod/vmi-system-data-ingest-1-8d7db6489-kdhbv           2/2     Running    1
 pod/vmi-system-data-ingest-2-699d6bdd9c-z7nzx          2/2     Running    0          5m21s
 pod/vmi-system-grafana-7947cdd84b-b7mks                2/2     Running    0          5m21s
 pod/vmi-system-kiali-6c7bd6658b-d2zq9                  2/2     Running    0          5m37s
-pod/vmi-system-kibana-7d47f65dfc-zhjxp                 2/2     Running    0          5m21s
+pod/vmi-system-opensearchDashboards-7d47f65dfc-zhjxp                 2/2     Running    0          5m21s
```
Comment (anders-swanson) [inline content/en/docs/customize/opensearch.md:94]: are these still column aligned?

**Review by anders-swanson — APPROVED** [review]

**Review by rosemarymarano — APPROVED** [review]
Yep, looks good now.

**Pull request merged**